builds/posix/make.rules

```diff
@@ -96,15 +96,18 @@ $(OBJ)/%.cpp: $(SRC_ROOT)/%.epp
 
 $(OBJ)/%.o: $(SRC_ROOT)/%.c
 	$(CC) $(WCFLAGS) -c $(firstword $<) -o $@
```

Comment:
In that case this should better be done in OS-specific manner 
BTW, looking at:
https://www.freebsd.org/cgi/man.cgi?sed%281%29 
I see that -i is present